example/item-one-image-archivespace/item-one-image-archivespace.csv

```diff
@@ -1,3 +1,3 @@
-sourceSystem,repository,collectionId,parentId,id,level,title,creator,dateCreated,uniqueIdentifier,dimensions,languages,subjects,usage,license,linkToSource,access,format,dedication,description,modifiedDate,thumbnail,filePath,fileId,sequence,collectionInformation,fileInfo,classification,workType,medium,artists,digitalAssets,width,height,etag,md5Checksum
+sourceSystem,repository,collectionId,parentId,id,level,title,creator,dateCreated,uniqueIdentifier,dimensions,languages,subjects,copyrightStatus,copyrightStatement,linkToSource,access,format,dedication,description,modifiedDate,thumbnail,filePath,fileId,sequence,collectionInformation,fileInfo,classification,workType,medium,artists,digitalAssets,width,height,etag,md5Checksum
```

Comment:
This list of fields is slightly updated now.  As per our discussion a few weeks ago, I've removed etag from my csv creation.  Also, I added a couple of additional fields at the end.

Reply:
Does @rdoughty know this?

Reply:
do you have a list of the added fields?

Reply:
@jonhartzler yes, I'm using md5Checksum